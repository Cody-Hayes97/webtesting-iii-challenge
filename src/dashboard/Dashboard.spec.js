import React from "react";
import * as rtl from "@testing-library/react";
import Dashboard from "./Dashboard";
import "@testing-library/jest-dom/extend-expect";

test("shows the controls and display", () => {
  const wrapper = rtl.render(<Dashboard />);

  expect(wrapper.asFragment()).toMatchSnapshot();
});
