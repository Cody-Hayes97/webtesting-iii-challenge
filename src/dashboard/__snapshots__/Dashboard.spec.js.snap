// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows the controls and display 1`] = `
<DocumentFragment>
  <div
    class="display panel"
  >
    <div
      class="led green-led"
      data-testid="isLocked"
    >
      Unlocked
    </div>
    <div
      class="led green-led"
      data-testid="isClosed"
    >
      Open
    </div>
  </div>
  <div
    class="controls panel"
  >
    <button
      class="toggle-btn"
      data-testid="unlock-btn"
      disabled=""
    >
      Lock Gate
    </button>
    <button
      class="toggle-btn"
      data-testid="open-btn"
    >
      Close Gate
    </button>
  </div>
</DocumentFragment>
`;
